src/deployable_example.cpp

```diff
@@ -789,14 +716,13 @@ void loop () {
 
```

Comment:
Immediately above is a return statement without a esp_camera_fb_return call. Using the frameLock class should change the behavior there as we now will implicitly call esp_camera_fb_return in the class destructor